app/routes/api/release.js

```diff
@@ -96,9 +95,9 @@ router.post('/:slug', function(req, res)
 			// Or else update the game defaults
 			else
 			{
-				_.assign(
+				Object.assign(
 					game.capabilities, 
-					_.cloneDeep(values.capabilities)
+					values.capabilities
```

Comment:
This could be a issue, as I can't remember if Object.assign does a deep clone or not, but will have to do some testing to make sure

Reply:
Good point. I think `capabilities` is typically a flat object, so we should be okay. However, we should still clone it with `Object.assign({}, values.capabilities)`.